spec/buftok_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'filewatch/buftok'
+
+describe FileWatch::BufferedTokenizer do
+
+  context "when using the default delimiter" do
+    subject { FileWatch::BufferedTokenizer.new }
+
+    it "splits the lines correctly" do
+      message = "hello\nworld\n"
```

Comment:
actually I'd prefer to have the message string in the assertion itself:

```ruby
expect(subject.extract("hello\r\nworld\r\n")).to eq ["hello", "world"]
```

this makes reading the test way easier